app/views/_container-statuses.html

```diff
@@ -0,0 +1,72 @@
+<!--
+  Expects the following variables:
+  containerStatuses
+  startDebugTerminal (optional)
+  detailed (optional)
+ -->
+
+<div ng-if="detailed">
+  <h4 row>
+    <span><i class="pficon" ng-class="{'pficon-ok': !initContainersFailed}"></i></span>
+    <span flex>
+      <span ng-if="!initContainersFailed"> &nbsp;{{podContainerInitStatuses.length}}</span>
+      <ng-pluralize count="podContainerInitStatuses.length"
+                    when="{'1': 'Init Container','other': 'Init Containers'}">
+      </ng-pluralize>
+      <span ng-if="!initContainersFailed">completed successfully</span>
+    </span>
+    <a class="page-header-link" href="" ng-click="toggleInitContainer()" ng-if="!expandInitContainers">Show Details</a>
+    <a class="page-header-link" href="" ng-click="toggleInitContainer()" ng-if="expandInitContainers">Hide Details</a>
+  </h4>
+
+  <div class="pod-template-block"
+       ng-if="expandInitContainers"
+       ng-repeat="containerStatus in podContainerInitStatuses | orderBy:'name'" >
```

Comment:
The init containers are run in order, so order is important. We shouldn't reorder them.